cmd/appmaker.go

```diff
@@ -93,7 +94,79 @@ func CreateAppCmd(appsAPIService AppsAPIService) *cobra.Command {
 	return appmakerCmd
 }
 
-//FetchAppCmd returns command to fetch apps of a user
+// LocalAppCmd returns command to deploy application from local
+func LocalAppCmd(appsAPIservice AppsAPIService) *cobra.Command {
+	var localAppCmd = &cobra.Command{
+		Use:   "local [NAME] [PATH]",
+		Short: "Deploy an application from local",
+		Args:  cobra.MaximumNArgs(2),
+		Run: func(cmd *cobra.Command, args []string) {
+			var (
+				err         error
+				language    string
+				application openapi.Application
+			)
+
+			if gctltoken == "" {
+				gctltoken, err = middlewares.SetToken(client)
+				if err != nil {
+					cmd.Print(err)
+					return
+				}
+			}
+
+			if len(args) != 2 {
+				cmd.Println("error: incorrect number of arguments, usage: gctl create local [NAME] [PATH]")
+				return
+			}
+			appName, pathToApplication := args[0], args[1]
+
+			if !middlewares.ValidateName(appName) {
+				cmd.PrintErr("App Name should have only alphanumeric characters, lowercase alphabets and should be of length 3-40.")
+			}
+
```

Comment:
add a middleware to check if path given is correct or not... user might give relative path not absolute, etc